4 login welcome sign in

tkomm4 opened the pull request:
Created a Welcome Page, Admin Login Page, and User Login Page

LoginWelcomePage:
LoginWelcomePage.module.css
LoginWelcomePage.tsx

AdminLoginPage:
LoginSigninPage.module.css
LoginSigninPage.tsx

UserLoginPage:
LoginSigninPage.module.css
LoginSigninPage.tsx

<img width="1280" alt="image" src="https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/103841340/473b2ed1-f07c-4a68-b46e-8f308e9a230b">

<img width="1280" alt="image" src="https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/103841340/832c2556-af8d-4c7d-978a-bbea89d385b8">

<img width="1280" alt="image" src="https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/103841340/dbcaa431-04bd-4167-af39-5c9bf60ce01a">

mkb02 committed:
created files and backgrounds, in addition to adding pictures for login pages

mkb02 committed:
added white rectangle on left side of pages for welcome/login

mkb02 committed:
Merge branch 'main' of https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging into 4-login-welcome-sign-in

tkomm4 committed:
Created welcome and login page, still some features to be implemented

tkomm4 committed:
finished welcome page. missing the show-password component & switching between user and admin log in using props component

tkomm4 committed:
some minor changes

tkomm4 committed:
This has all the features except for the eyeicn

tkomm4 committed:
added the admin sign in page

tkomm4 committed:
changed the names of some files to improve readability

tkomm4 committed:
reverting some naming changes

yieb commented on jaba/src/assets/signin.png:
Switch to the blue-toned image that the designers updated on the Figma

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+.splitScreen {
+    height: calc(100vh);
```
I believe you can just do 100vh here since you just have a constant value without mixing different measurements

tkomm4 replied:
done

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+    left: 125px;
+    border-radius: 10px;
+    border: 0.5px;
+    background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
```
Can't you just do "background: white" here?

tkomm4 replied:
done

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+}
+
+.rightImage {
+    width: 50%;
```
Can you update to 60% for height/width?

yieb replied:
And do the same for the other page as well

tkomm4 replied:
done

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+}
+
+.labelProperties {
+    font-family: 'Poppins', ;
```
Change to all instances of 'Inter'

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+}
+
+.placeholderProperties {
+    font-family: 'Poppins', ;
```
Change to all instances of 'Inter'

yieb commented on jaba/src/pages/LoginWelcomePage/LoginWelcomePage.tsx:
```diff
@@ -0,0 +1,42 @@
+import styles from './LoginWelcomePage.module.css';
+import Welcome from '../../assets/welcome.png';
+
+<head>
```
You can remove this and just use the custom fonts in the fonts folder

yieb commented on jaba/src/pages/LoginWelcomePage/LoginWelcomePage.module.css:
```diff
@@ -0,0 +1,115 @@
+}
+
+.signInAdminButton {
+    /* font-family: 'DM Serif Text', serif; doesnt work? */
```
It should be working? You have the fonts/ folder in your branch

yieb commented on jaba/src/pages/LoginWelcomePage/LoginWelcomePage.module.css:
```diff
@@ -0,0 +1,115 @@
+    padding-top: 100px;
+    margin-bottom: 35px;
+}
+.signInUserButton {
```
Can you style some properties of .signInUserButton and .signInAdminButton at once? They share a good number of CSS properties

tkomm4 replied:
i feel like it's better to have them seperate? just in case we want to change one in the future, but also because they're different colors so they'd hve to have different catagories regardless

yieb replied:
I'm going to still style some properties at once to reduce CSS code. Your point is still valid but I think it would be better to jut merge some styling to reduce redundancy since its just styling on two buttons and if the style on one changes, it sholdn't be too difficult to adapt still

yieb commented:
![image](https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/assets/52669318/7882d5f9-7ea2-4891-a7a2-166a9b477fd1)

This is what I see when I load up the SignInAdmin page. The info on the left seems to be larger than my screen. Additionally, the Forgot Password? text is not right-aligned with the textbox.

yieb commented on jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css:
```diff
@@ -0,0 +1,163 @@
+}
+
+.alignWelcome, .alignWelcomeBlurb {
+    margin-bottom: -25px; /* Adjust this value as needed to control the spacing */
```
Using negative margin is generally considered bad practice as it makes the code more complex and difficult to understand. You can definitely style page the same way by only using positive values for margin/padding. It would be helpful if you could update that

yieb replied:
Additionally, consider using units like percentages or  vh/vw or rem for spacing / sizing properties instead of px since this is supposed to help make the page more responsive to different sizes

tkomm4 committed:
added some of the suggestions eric made about readibility and changed some the main images in each page

yieb committed:
Merge branch 'main' into 4-login-welcome-sign-in

yieb committed:
Fix various styling issues on Welcome Page

- Remove link to stylesheet in .tsx file (switch to using the custom
  fonts in the fonts/ folder)
- Remove unneeded use of calc() and gradient()
- Adjust image size

yieb committed:
Convert styling to avoid negative margins and px

yieb committed:
Add eye icon on Password field + switch to unicon

yieb commented:
Merging in to perform demo, still issues with how it looks + should still update so form does not use negative margins and padding to control layout

yieb committed:
Delete jaba/src/assets/invisibleeye.svg

yieb commented:
Could probably combine SignIn forms for user/admin later - or at least make them share the same CSS module file
